src/hotspot/os/linux/cgroupV2Subsystem_linux.hpp

```diff
@@ -42,23 +39,26 @@ class CgroupV2Controller: public CgroupController {
   public:
     CgroupV2Controller(char* mount_path,
                        char *cgroup_path,
-                       bool ro) :  _mount_path(os::strdup(mount_path)),
-                                   _cgroup_path(os::strdup(cgroup_path)),
-                                   _read_only(ro),
+                       bool ro) :  _read_only(ro),
                                    _path(construct_path(mount_path, cgroup_path)) {
+      _cgroup_path = os::strdup(cgroup_path);
+      _mount_point = os::strdup(mount_path);
     }
     // Shallow copy constructor
     CgroupV2Controller(const CgroupV2Controller& o) :
-                                            _mount_path(o._mount_path),
-                                            _cgroup_path(o._cgroup_path),
                                             _read_only(o._read_only),
                                             _path(o._path) {
+      _cgroup_path = o._cgroup_path;
+      _mount_point = o._mount_point;
```

Comment:
I think the constructors should move to the C++ file.

Reply:
Done.